share/scripts/inverse/iie.py

```diff
@@ -0,0 +1,1005 @@
+#!/usr/bin/env python3
+# """Multi purpose script for HNC inverse methods"""
+#
+# Copyright 2009-2017 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Symbols:
+# g: RDF
+# U: potential
+# dU: potential update (U_{k+1} - U_k)
+#
+# suffixes:
+# _cur: current (of step k if currently doing iteration k)
+# _tgt: target
+# _ce: core_end (where RDF becomes > 0)
+# _co: cut_off
+# _sc: single_component
+#
+# prefixes:
+# ndx_: index
+
+
```

Comment:
Some unittests using pytest would be really nice https://blog.esciencecenter.nl/a-quick-and-dirty-pytest-cookbook-58051a2980c3